apps/nextjs/src/app/(dashboard)/dashboard/page.tsx

```
@@ -1,5 +1,20 @@
-import DashboardContentSkeleton from "./_components/dashboard-content-skeleton";
+import { DashboardContentBody } from "./_components/dashboard-content-body";
+import { DashboardContentSection } from "./_components/dashboard-content-top-section";
 
 export default function DashboardProfiles() {
-  return <DashboardContentSkeleton />;
+  return (
+    <section
+      role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)